templates/files/swift5.stencil

```diff
@@ -0,0 +1,68 @@
+// swiftlint:disable all
+// Generated using SwiftGen - https://github.com/SwiftGen/SwiftGen
```

Comment:
Is that the proper type of dash that you used here (em dash vs en dash vs hyphen)? Not totally sure as it's hard to check from iPhone, but I think that could be the cause of your `file_header` violations 😅

```suggestion
// Generated using SwiftGen — https://github.com/SwiftGen/SwiftGen
```

Reply:
That would be nuts, but no I believe it's the right dash. I just ran the generator (I had written them by hand) and the header remained the same

Reply:
Tbh now seeing how the GitHub suggestion is rendered, highlighting the difference before/after, I'm pretty sure that's the reason tho. The one you used seems to be a hyphen while the one I put in my GitHub suggestion seems to be 1 or 2 pixels wider and be an em dash (which is the more grammatically correct one in this context anyway).

It's not surprising that running the generator scheme didn't change anything as you also used the hyphen instead of em dash in the template itself (`.stencil`) 😛

Reply:
Ah. Haha, yes I see it now. Thanks :)

Reply:
Cool! Pfeew, that was a tricky and super subtle one tho 😅